python/pyspark/util.py

```diff
@@ -18,6 +18,8 @@
 
 __all__ = []
 
+import sys
+
```

Comment:
nit: remove an extra line.

Reply:
looks like the newline is required by lint. :)

Reply:
Oh, I see. Thanks!